packages/bitcoin/redux/reducers.js

```diff
@@ -8,12 +8,22 @@ import initialState from './initialState'
 
 const mutations = {
 
-  [ActionsTypes.BITCOIN_CREATE_WALLET]: (state, { address }) => ({
-    ...state,
-    list: {
-      address,
-    },
-  }),
+  [ActionsTypes.BITCOIN_CREATE_WALLET]: (state, { parentAddress, address }) => {
+    let list = Object.assign({}, state.list)
```

Comment:
Why not just return with no additional variables?

Reply:
I usually use this variant of copying object